app/utils/market.ts

```diff
@@ -114,16 +114,15 @@ export const marketIsQuotePair = (
 
 export const marketIsPartOfType = (
```

Comment:
General rule of thumb: more than 2 arguments within a function -> make the argument an object.